charters/PipelineExecutionService/charter.md

```diff
@@ -0,0 +1,63 @@
+# [Pipeline Execution Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Pipeline Execution Service executes data processing pipelines containing analysis tasks performed on scientific data in the Data Coordination Platform (DCP). The outputs of these pipelines are used for scientific analysis and interpretation.  
+
+## Objectives
+
+The Pipeline Execution Service runs pipelines reliably and at-scale for the DCP. The team implementing the service also operates the pipeline infrastructure for the Human Cell Atlas project. 
+
+## Definitions
+
+__pipeline:__ A collection of one or more functional tasks that operate on input data and, from that, make an output that is transformed input data, or derive features often used to interpret the input data. In a high throughput setting, these tasks are often automated to be performed in a batch setting.
+
+## In-scope
+
+### Operations
+
+* Operating the instance of the HCA DCP Data Processing Service
+* Initiation and management of the execution of all data processing pipelines that run on scientific data in the HCA DCP  
+* Establishing and operating automated practices around engineering testing of pipelines executed in the HCA DCP Data Processing Service   
+
+### Documentation
+
+* Creating and outputting metadata on the operations applied to the data in accordance with the HCA DCP Metadata Schema  
+* Documentation on the Data Processing Service and guides on using the service  
+* Creating documentation and scripting around the deployment of the Data Processing Service
+
+## Milestones and Deliverables
+
+__2018:__ Create operational and secure infrastructure, add ability to throttle and queue  
```

Comment:
optional: ability to throttle and queue the executions of data processing pipelines/workflows

Reply:
added